src/test/java/com/bouncestorage/chaoshttpproxy/ChaosHttpProxyTest.java

```diff
@@ -68,10 +75,27 @@
     private HttpBin httpBin;
     private HttpClient client;
 
+    private ChaosConfig chaosConfig;
+
     @Before
     public void setUp() throws Exception {
-        proxy = new ChaosHttpProxy(proxyEndpoint,
-                Suppliers.ofInstance(Failure.SUCCESS));
+        chaosConfig = new ChaosConfig() {
```

Comment:
I wanted to isolate the pieces under test. Under normal circumstances, I would have added a mock here but, since this is a simple interface and test config, and there's no mocking going on otherwise, I just created a simple version for the test.